nixos/modules/services/system/uptimed.nix

```diff
@@ -21,12 +21,21 @@ in
   };
 
   config = mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.uptimed ];
+
+    systemd.tmpfiles.rules = [
+      "d '${stateDir}' 0775 uptimed uptimed"
```

Comment:
Since it's always in /var/lib, you could also set `StateDirectory = "uptimed"` in the service instead